data/README.md

```diff
@@ -0,0 +1,43 @@
+# Data sources
+
+## data/target_scan_8 (downloaded on 1.6.2024)
```

Comment:
Header levels are confusing here, they are not consistent.
I would do level 2 description - so in this case it would be "Predicted Targets context++ scores (default predictions) - (17.67 MB)".
If data is already present in the GitHub repository, mention it in the section. Something like:
```markdown
## Predicted Targets context++ scores (default predictions)
Source: [https://www.targetscan.org/cgi-bin/targetscan/data_download.vert80.cgi](https://www.targetscan.org/cgi-bin/targetscan/data_download.vert80.cgi)
Destination: data/target_scan_8
Downloaded on: 1.6.2024
Source update on: 23.5.2022
Size: 17.67 MB
Description: Context++ scores for predicted (conserved) targets of conserved miRNA families
Columns: Gene ID, Gene Symbol, Transcript ID, Species ID, miRNA, Site type, UTR start, UTR end, context++ score, context++ score percentile, weighted context++ score, weighted context++ score percentile, predicted relative KD
Number of rows: 1,397,979
```
Please update description for all sources

Reply:
Done